README.md

```diff
@@ -341,7 +349,9 @@ The following env variables can be used
 | EQ_WERKZEUG_LOG_LEVEL                     | INFO                  | The default logging level for werkzeug (defaults to 'INFO' for local development)             |
 | EQ_SESSION_TIMEOUT_SECONDS                | 2700 (45 mins)        | The duration of the flask session                                                             |
 | EQ_PROFILING                              | False                 | Enables or disables profiling (True/False) Default False/Disabled                             |
-| EQ_UA_ID                                  |                       | The Google Analytics ID                                                                       |
+| EQ_GTM_ID                                 |                       | The Google Tag Manger ID - Specifies the GTM account                                          |
+| EQ_GTM_AUTH                               |                       | The Google Tag Manger Auth - Ties the GTM container with the whole enviroment                 |
+| EQ_GTM_PREVIEW                            |                       | The Google Tag Manger Preview - Specifies the environment                                     |
```

Comment:
It would be clearer to expand `GTM` to `GOOGLE_TAG_MANAGER`

Reply:
Done